notebooks/docker-compose.yml

```diff
@@ -21,3 +21,9 @@ services:
     cap_add:
       - SYS_PTRACE  # to enable gdb debugging
     tty: true
+    ports:
+      # Host will chose a random available port. To find the port allocated, run
+      # docker-compose -f notebooks/docker-compose.yml port notebooks 8888
+      - 127.0.0.1::8888
```

Comment:
Do we need the first bit? It took me a while to understand what was going on.
This would be clearer to me:
```
ports:
  - "8888"
```
Also to find port the quickest way is `docker ps` right? It auto filters to your own containers.

Reply:
If you don't include the `127.0.0.1` then it will expose the port on public interfaces, which seems like an insecure default to have. I can add a comment explaining what it does.

Reply:
Oh okay thanks, good to know.